catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/bots/BotsResource.java

```diff
@@ -61,7 +60,6 @@
   private final BotsRepository dao;
   private final Authorizer authorizer;
 
-  @Inject
```

Comment:
Can be safely removed.

These resources are instantiated by `createResource` method in `CollectionRegistry.java` through parameterized constructor.

![image](https://user-images.githubusercontent.com/5906984/148701018-b6875c15-1a3b-45e7-9ece-ca65375fcff5.png)